File: ws/Perez/U1/WS08CDToCode/ChickenFarmSimulator.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.farmsimulator.view;

import ec.edu.espe.farmsimulator.model.Chicken;
import java.util.Date;

/**
 *
 * @author Carlos Pérez, The Java Bandits, DCCO-ESPE
 */
public class ChickenFarmSimulator {
    public static void main(String[] args){
        System.out.println("Hello chickens from Carlos Perez \nChicken Farm Simulator 2K24 V 0.1");
        
        Chicken chicken = new Chicken(1, "Lucy", "Black", 1, true, new Date());
           
        System.out.println("Chicken --> " + chicken);
        
        int id = 2;
        String name = "Maruja";
        String color = "white";
        int age = 0;
        boolean molting = false;
        Date bornOnDate = new Date();
        
        chicken = new Chicken(id, name, color, age, molting, bornOnDate);
        System.out.println("Chicken --> " + chicken);
    }
}

File: ws/travez/u1/ws08ChickenFarmSimulator/Python/ChickenFarmSimulator.py
from datetime import date
from Chicken import Chicken

def main():
    print("Hello chickens from Ismael Travez \nChicken Farm Simulator 2K24 V 0.1")

    chicken = Chicken(1, "Lucy", "Black", 1, True, date.today())
    print(f"Chicken --> {chicken}")

    id = 2
    name = "Maruja"
    color = "white"
    age = 0
    molting = False
    born_on_date = date.today()

    chicken = Chicken(id, name, color, age, molting, born_on_date)
    print(f"Chicken --> {chicken}")

if __name__ == "__main__":
    main()
